### DISCUSSION TITLE
gRPC with cypress

### DISCUSSION THREAD

**Discussion by valiaPi** [cypress-io/cypress#17768] [Questions and Help]
Hello,
I am new to cypress and I am trying to write an e2e test where I also use gRPC. Although when running my script standalone it works fine, when I use grpc with cypress I get this error 
_Error: Can't walk dependency graph: Cannot find module 'http2' from 'C:\Users\panagiot-1\Source\Repos\LabXConnected.DataIntegrity\specs\audittrail-e2e\node_modules\@grpc\grpc-js\build\src'_
Any opinions on how to solve this?
Thanks a lot

**Comment by valiaPi** [comment]
Solved it by using cy.task()

**Reply by Parsa-Sedigh** [reply]
I know it's late, but do you mind sharing your code if you still have it? Are you using cy.intercept() to stub the grpc response? 
I'm using cypress-protobuf plugin to stub the grpc response with a fixture that is encoded with the protobuf file, but no luck

**Reply by tolerious** [reply]
cy.intercept() can only intercept unary call, and can not intercept server stream call. How did you solve it?
